tests/test_callbacks.py

```diff
@@ -35,14 +37,17 @@ def judge(self, prompts, completions, shuffle_order=True):
 class TrainerWithRefModel(Trainer):
     # This is a dummy class to test the callback. Compared to the Trainer class, it only has an additional
     # ref_model attribute
-    def __init__(self, model, ref_model, args, train_dataset, eval_dataset, tokenizer):
+    def __init__(self, model, ref_model, args, train_dataset, eval_dataset, processing_class):
         super().__init__(
-            model=model, args=args, train_dataset=train_dataset, eval_dataset=eval_dataset, tokenizer=tokenizer
+            model=model,
+            args=args,
+            train_dataset=train_dataset,
+            eval_dataset=eval_dataset,
+            processing_class=processing_class,
         )
         self.ref_model = ref_model
 
 
-@require_wandb
```

Comment:
this one doesn't require wandb. I suspect that the error was not detected because this test was skipped while it shouldn't